addons/l10n_ar_withholding/models/account_tax.py

```diff
@@ -1,13 +1,19 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class AccountTax(models.Model):
 
     _inherit = 'account.tax'
 
     l10n_ar_withholding_payment_type = fields.Selection(
-        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type')
+        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type',
+        compute="compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
```

Comment:
```suggestion
        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
```
This does not seem to bring a lot of value @vbe-odoo

Reply:
Not a lot indeed, but already an improvement/fix I guess, so it won't hurt to have it if technically not too complicated

Reply:
Hi @vbe-odoo @jco-odoo 
Thank you for the review

I have updated the PR with the suggested changes
Let me know if there is anything else needed in order to merge this pr

Best Regards